luacheck-0.23.0-1.rockspec

```diff
@@ -0,0 +1,83 @@
+package = "luacheck"
+version = "0.23.0-1"
+source = {
+   url = "git+https://github.com/mpeterv/luacheck.git",
```

Comment:
Let's update our fork: https://github.com/tarantool/luacheck

Reply:
Ok. Actually we have luacheck in our rocks (https://github.com/tarantool/rocks/pull/73)

So, let's merge tagged version. But please add "Closes #21" to commit message: https://github.com/tarantool/rocks/issues/21